native_client/client.js

```diff
@@ -3,13 +3,20 @@ const Sox = require('sox-stream');
 const Ds = require('deepspeech');
 const MemoryStream = require('memory-stream');
 
+const BEAM_WIDTH = 500;
+const LM_WEIGHT = 2.15;
+const WORD_COUNT_WEIGHT = -0.10;
+const VALID_WORD_COUNT_WEIGHT = 1.10;
+
 var audioStream = new MemoryStream();
 Fs.createReadStream(process.argv[3]).
   pipe(Sox({ output: { bits: 16, rate: 16000, channels: 1, type: 'raw' } })).
   pipe(audioStream);
 audioStream.on('finish', () => {
   audioBuffer = audioStream.toBuffer();
-  var model = new Ds.Model(process.argv[2], 26, 9, process.argv[4]);
+  var model = new Ds.Model(process.argv[2], 26, 9, process.argv[4],
```

Comment:
nit: since we introduce some `const` above, we should move the `26` and `9` values as `const` as well